src/data/moves/move-attrs/copy-stats-attr.ts

```diff
@@ -35,4 +37,37 @@ export class CopyStatsAttr extends MoveEffectAttr {
 
     return true;
   }
+
+  /**
+   * @returns (+0.5) for each stat stage the user would gain from this effect against the given
```

Comment:
```suggestion
   * @returns (+1) for each `2` stat stages the user would gain from this effect against the given
```
Since you are `floor`ing the result, it won't actually return fractional values.